README.md

```diff
@@ -69,12 +69,9 @@ python src/manage.py test
 
 ## GitHub Developer Sensitive Data
 
-The `settings.py` file uses sensitive data to work with the GitHub API. For that reason, sensitive data is not checked into source. Due to that lack of data, the following error could rise when running the app:
+The `settings.py` file uses sensitive data to work with the GitHub API. For that reason, sensitive data is not checked into source. Due to that lack of data, the License submittal feature and its API won't be able to run as they require the user credentials in order to access the GitHub API. So, the user is supposed to enter their credentials in the `src/src/secret.py ` file in order to ensure proper functioning of the tool.
```

Comment:
> Due to that lack of data, the License submittal feature and its API won't be able to run ...

Instead of `License submittal feature` it should be `some features of SPDX Online Tools` because credentials are also used by the XML Editor.